src/App.tsx

```diff
@@ -1,24 +1,193 @@
+      title: trimmedTitle,
+      completed: false,
+    };
+
+    setTempTodo({
+      id: 0,
+      ...data,
+    });
+
+    setStatusResponse(true);
+
+    todosService
+      .createTodo(data)
+      .then((newTodo) => {
+        setTitle('');
+        setTodos((currentTodos) => [...currentTodos, newTodo]);
+      })
+      .catch(() => {
+        setError('Unable to add a todo');
+      })
+      .finally(() => {
+        setTempTodo(null);
+        setStatusResponse(false);
+      });
+  }
+
+  const visibleTodos = filteredTodos(todos, status);
+  const handleFilterStatus = (todosStatus: Status) => (
+    setStatus(todosStatus));
+
+  const activeTodosCount = todos.filter(todo => !todo.completed).length;
+  const completedTodosCount = todos.filter(todo => todo.completed).length;
+
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          {activeTodosCount > 0 && (
+            <button
+              type="button"
+              className="todoapp__toggle-all active"
+              data-cy="ToggleAllButton"
+              aria-label="todoapp__toggle-all active"
+            />
+          )}
+
+          <TodoForm
+            title={title}
+            setTitle={(value) => setTitle(value)}
+            onSubmit={() => addTodo()}
+            statusResponse={statusResponse}
+          />
+        </header>
+
+        {!isLoading && (
+          <>
+            <TodoList todos={visibleTodos} deleteTodo={deleteTodo} />
+
+            {tempTodo && <TodoItem onDelete={deleteTodo} todo={tempTodo} />}
+
+            {!!todos?.length && (
+              <footer className="todoapp__footer" data-cy="Footer">
+                <span className="todo-count" data-cy="TodosCounter">
+                  {`${activeTodosCount} items left`}
+                </span>
+
+                <TodoFilter
+                  handleFilterStatus={handleFilterStatus}
+                  todosFilterStatus={status}
+                />
+
+                {completedTodosCount > 0 && (
+                  <button
+                    type="button"
+                    className="todoapp__clear-completed"
+                    data-cy="ClearCompletedButton"
+                  >
+                    Clear completed
+                  </button>
+                )}
+              </footer>
+            )}
+          </>
+        )}
+      </div>
+
+      {/* Notification is shown in case of any error */}
+      {/* Add the 'hidden' class to hide the message smoothly */}
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)